src/cogent3/core/annotation_db.py

```diff
@@ -56,6 +57,14 @@ def sqlite_to_array(data):
         result = numpy.frombuffer(data, dtype=int)
         dim = result.shape[0] // 2
         result = result.reshape((dim, 2))
+
+        warnings.warn(
+            "Old OS dependent file format detected. "
+            "Please update file format to latest version. "
```

Comment:
Change the second line to explicit advice `Update the file format using cogent3.core.annotation_db.update_file_format()`

Reply:
Will do!